docs/OV_Runtime_UG/Integrate_with_customer_application.md

```diff
@@ -33,26 +33,21 @@ Read the sections below to learn about each item.
        ...      
    ```
 
-2. **Include Inference Engine, nGraph and OpenCV libraries** in `project/CMakeLists.txt`  
-[OpenCV](https://docs.opencv.org/master/db/df5/tutorial_linux_gcc_cmake.html) integration is needed mostly for pre-processing input data and model representation in OpenVINO™ Runtime for more complex applications using [OpenVINO Model API](../OV_Runtime_UG/model_representation.md).
-   ``` cmake
-   cmake_minimum_required(VERSION 3.0.0)
-   project(project_name)
-   find_package(OpenVINO REQUIRED)
-   add_executable(${PROJECT_NAME} src/main.cpp)
-   target_link_libraries(${PROJECT_NAME} PRIVATE openvino::runtime)
-   ```
+2. **Include OpenVINO™ Runtime libraries** in `project/CMakeLists.txt`
+
+   @snippet snippets/CMakeLists.txt cmake:integration_example
 
-### Use Inference Engine API to Implement Inference Pipeline
+### Use OpenVINO™ Runtime API to Implement Inference Pipeline
 
-This section provides step-by-step instructions to implement a typical inference pipeline with the Inference Engine C++ API:   
+This section provides step-by-step instructions to implement a typical inference pipeline with the OpenVINO™ Runtime C++ API:
 
 ![ie_api_use_cpp]
-#### Step 1. Create Inference Engine Core 
 
-Use the following code to create Inference Engine Core to manage available devices and read network objects:
+#### Step 1. Create OpenVINO™ Runtime Core 
 
-@snippet snippets/Integrate_with_customer_application_new_API.cpp part0
+Use the following code to create OpenVINO™ Core to manage available devices and read network objects:
+
+@snippet snippets/src/main.cpp part0
 
 #### Step 2 (Optional). Configure Input and Output of the Model
```

Comment:
do we need to demonstrate this? we don't demonstrate any configuration of i/o actually here..

Reply:
Added reference to separate page